doc/OnlineDocs/contributed_packages/alternative_solutions.rst

```diff
@@ -88,6 +88,53 @@ Each ``Solution`` object contains information about the objective and variables,
    }
 
 
+Gap Usage Example
+-------------
+When we only want some of the solutions based off a tolerance away from optimal, this can be done using the ``abs_opt_gap`` parameter. This is shown in the following simple knapsack examples where the weights and values are the same.
```

Comment:
Not a hard requirement, but it would be good to wrap paragraphs to 88 characters (following the Black formatting standard we use in the code)